code/src/components/TaskList.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react'
+import { useSelector, useDispatch } from 'react-redux'
+import styled from 'styled-components/macro'
+
+import tasks from '../reducers/tasks'
+import EmptyStateAnimation from './EmptyStateAnimation'
+
+const TaskContainer = styled.div`
+    overflow: auto;
+    width: 100%;
+`
+
+const SingleTaskContainer = styled.div`
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-between;
+    margin-top: 4px;
+    margin-bottom: 8px;
+    border-bottom: 1px solid black;
+    word-break: break-word
+`
+
```

Comment:
A wrote the longest word ever and it worked like a charm, must be this setting you remembered right here 👍